platform/lifecycle_inputs.go

```diff
@@ -85,11 +86,12 @@ func NewLifecycleInputs(platformAPI *api.Version) *LifecycleInputs {
 	inputs := &LifecycleInputs{
 		// Operator config
 
-		LogLevel:    envOrDefault(EnvLogLevel, DefaultLogLevel),
-		PlatformAPI: platformAPI,
-		ExtendKind:  envOrDefault(EnvExtendKind, DefaultExtendKind),
-		UseDaemon:   boolEnv(EnvUseDaemon),
-		UseLayout:   boolEnv(EnvUseLayout),
+		LogLevel:         envOrDefault(EnvLogLevel, DefaultLogLevel),
+		PlatformAPI:      platformAPI,
+		ExtendKind:       envOrDefault(EnvExtendKind, DefaultExtendKind),
+		UseDaemon:        boolEnv(EnvUseDaemon),
+		InsecureRegistry: nil,
```

Comment:
Should this be `InsecureRegistries`?

Reply:
Uhm, in the RFC the flag is `--insecure-registry` so I tried to kept that.
Because if I changed it, we would have something like:
```
FlagInsecureRegistries(insecureRegistries *str.Slice) {
	flagSet.Var(insecureRegistries, "insecure-registry", "to use 'insecure' registries")
}
```
But it is kind of misleading because the name and the param are about insecureRegistries but the name of the param is `insecure-registry` which can cause confusion.
For this reason I kept InsecureRegistry on the input parts and then used insecureRegistries only in the image part.
What do you think?
Should I keep everything plural or singular? Should we change the parameter name?

Reply:
I'm gonna change the name and push it, so we can see how it looks like :)